_pages/docs/request-match-rules/templated-responses.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: single
+title: Templated Responses
+permalink: /docs/templated-responses
+sidebar:
+  nav: 'docs'
+classes: wide
+---
+
+Templated responses allow responses to be dynamically created by using Scriban's liquid template API. The responses can be derived from the response's body, and/or a random value using the [faker](https://github.com/bchavez/Bogus) API.
```

Comment:
can we redo this sentence and not mention the technologies we are using. We want to explain just the feature, not so much what technologies we are using to do this feature.

Reply:
Fixed